notebooks/notebooks/iclr2022/checkpoint-transfer.ipynb

```diff
@@ -15,7 +15,10 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.    fig, _ = plot(df, "adv-cp127only-s17550336-v600", "$\\texttt{cp127}$-trained adv.")
</pre><p>We use a later step for cp127 than the cp505 adversary? OK if true but a bit counterintuitive (takes longer to get good performance for cp127 adversary than cp505?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I ran them for the same amount of wall clock time until both had win rates >99%. Since I went by wall clock time, we expect the cp127 games to have more throughput due to using a smaller victim network. I didn't check that both runs used the same GPU type though so it's possible that the compute usage was still unequal